src/Raven.Server/Web/System/Processors/Backups/BackupDatabaseHandlerProcessorForGetPeriodicBackupStatus.cs

```diff
@@ -21,13 +21,17 @@ public BackupDatabaseHandlerProcessorForGetPeriodicBackupStatus([NotNull] Reques
     public override async ValueTask ExecuteAsync()
     {
         var name = RequestHandler.GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+        var type = RequestHandler.GetStringQueryString("type", required: false) ?? nameof(StatusType.Cluster);
```

Comment:
does anyone use this? if not can we remove this?
does this somehow affect the studio?

Reply:
it might be useful for debugging? should we include this info in the debug package?

Reply:
My assumption is that Stav initially did this for debugging purposes.
I think we should leave it.
I've created a separate issue for including this data in the debug package: [RavenDB-24681](https://issues.hibernatingrhinos.com/issue/RavenDB-24681)